aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/CodegenExtensions.kt

```diff
@@ -27,3 +27,14 @@ internal val AuthProvider.userPoolProviderName: String
             else -> providerKey
         }
     }
+
+internal val AuthProvider.identityPoolProviderName: String
```

Comment:
`identityPoolProviderName` is confusing as they are not identity pool providers. I think we can call them token providers or identity providers.

Reply:
Ok, I had named to match iOS implementation. I'll change to identityProviderName to be similar still.